### PR TITLE
Meat Lab Ghoul Nests

### PR DESCRIPTION
**Pull request by John-Candlebury**
#### Summary
Content "Create unique content for the meat lab."

#### Purpose of change
Quite the title.

Add some unique chunks that directly relate to the monsters in:  #72969.  Meant to spawn both the creatures and items/mechanics directly related to them.

More incoming as the linked PR develops.

#### Describe the solution

Uses the same chunk-parametric pallete system that powers the normal and meat microlab variants.

### CODE REVIEW

**Comment by KHeket** [comment]
Can you also remove breathers mentioned in #71349 because of them the meat labs became a most fps death places?

**Comment by PatrikLundell** [comment]
Minor linguistic clarification: A pallet is something used for moving goods around, while a palette is what painters use when painting. Probably not much risk for confusion, as pallets aren't used in the game, I think. I've seen several cases of the wrong word being used, hence this comment.

**Comment by John-Candlebury** [comment]
> Can you also remove or highly reduce number of breathers mentioned in #71349 because of them the meat labs became a most fps death places?

Gave the issue a read and it seems the bug is related to monster vision, they are processing vision even when their definitions state they should be blind. Not able to fix this, and I rather not simply hide the underlying bug by making the breathers not spawn. Sorry.

**Comment by Maleclypse** [comment]
Hey John, if you don't mind I'm going to look at getting this complete.

File: data/json/furniture_and_terrain/furnirure-flesh.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+[
+  {
+    "type": "furniture",
+    "id": "f_cartilage_strands",
+    "name": "cartilaginous strands",
+    "description": "Thin cartilage webbing stretches between the overgrowth, forming a sticky, elastic barrier.",
+    "symbol": "#",
+    "color": "white",
+    "move_cost": -1,
```
Comment (Maleclypse) [inline data/json/furniture_and_terrain/furnirure-flesh.json:9]:
```suggestion
    "move_cost_mod": -1,
```

**Comment by Night-Pryanik** [comment]
Closing as stale. If you wish to continue working on this, ping me to reopen.

**Comment by John-Candlebury** [comment]
> Hey John, if you don't mind I'm going to look at getting this complete.

Kinda lost track of this in between other stuff, ltmk if you still want to work on it or if I add it to my pile of revisit later features.

**Pull request closed without merging**